American Whitewater/Helpers/AWApiHelper.swift

```diff
@@ -0,0 +1,331 @@
+
+// This was the web-based / non-CoreData version
```

Comment:
If this file isn't used anymore, maybe we can delete?

Reply:
Deleted